src/Persistence/Array_.php

```diff
@@ -81,10 +81,8 @@ public function add($m, $defaults = [])
      * @param Model  $m
      * @param mixed  $id
      * @param string $table
-     *
-     * @return array|false
      */
-    public function load(Model $m, $id, $table = null)
+    public function load(Model $m, $id, $table = null): array
```

Comment:
Looks like $table can be only string, so we can set type here. I doubt that somebody will set table name as integer.

Reply:
This PR probably does not solve this :)

Reply:
@DarkSide666 I checked now - yes, `$table` must be scalar (it is used as an array key later) and for these reasons `string` is the only valid type. But SQL does not have this parameter - do we need table at all (any strong usecase)? We can remove it, user can still set it using `Model::table` then.

Reply:
Maybe some use-case related to audit (historical) table.
Can't imagine anything else now which can't be solved by setting Model::table.